.github/workflows/sync-rest.yml

```diff
@@ -22,6 +22,30 @@ env:
   GIT_EMAIL: "66069679+exercism-bot@users.noreply.github.com"
 
 jobs:
+  open-tracking-issue:
+    name: Open tracking issue
+    runs-on: ubuntu-20.04
+
+    outputs:
+      issue-url: ${{ steps.open-tracking-issue.outputs.result }}
+    
+    steps:
+      - name: Open tracking issue
+        id: open-tracking-issue
+        uses: actions/github-script@a3e7071a34d7e1f219a8a4de9a5e0a34d1ee1293
+        with:
+          result-encoding: string
+          script: |
+            const issue = (await github.issues.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: `[Tracking] ${context.workflow} - ${context.eventName} - ${context.actor} - ${context.sha}`,
```

Comment:
We probably need to do something to accommodate the `pusher` field in the repo_dispatch event, right?

Reply:
I was planning to add pusher handling later in a follow-up so we can merge this without waiting for the website PR. It'd only have to be changed in the file that handles the repo dispatch events. For the non-repo-dispatch events, `actor` is correct, it's whoever triggered the build either by merging a PR in this repo or via manual workflow dispatch.

In case of a repo dispatch, we probably want this to be 

```
[Tracking] ${context.workflow} - ${context.eventName} - REPO - PUSHER - (PUSH_SHA?)
```

Reply:
Perfect.